### ISSUE TITLE
Some experiment doubts

### ISSUE THREAD

**Issue by lzhnb** [Jia-Research-Lab/PointGroup#24]
Thanks for your amazing PointGroup.

And I've done some experiments about it, I have some doubts:

1. I test your pretrain model on validation set, the performance for input data with dataAugment [35.2/57.4/71.8] is higher than origin input without dataAugment (all parameters set `False`)[33.6/55.1/70.0] (for [mAP/mAP50/mAP25]).

2. I decrease the length of backbone (`[m, ..., 7*m] -> [m, ..., 5*m]`), and set `m=32` replace `m=16`, which improved performance slightly.

3. The training time is verbose, so sometimes I test the performance of each epoch saved frequently, (your save step is `16` epoch, but it also saves the latest epoch), I found there would be some performance fluctuations between two adjacent epochs, sometimes the fluctuations is large(around 2 percents), even the for some saved epochs (like `320\368`, `320\384`), the latter epoch's performance maybe worse. Your tensorboard loss curve shows the ScoreNet loss will increase in some epoch[issue#7](https://github.com/Jia-Research-Lab/PointGroup/issues/7).

4. Your pretrain model's performace can correspond to the results in your ablation study, but the performance improvement between test set and validation set is huge, I've found the improvement between validation set and test set is common in other methods, but around 7 percent improvement about AP50 is crazy, I think you may do some fine-tuning before report your test set result as you say in [issue#9](https://github.com/Jia-Research-Lab/PointGroup/issues/9). Could you please share some fine-tuning details?

5. Is the elastic distortion necessary? With this distortion and your origin setting, it is difficult to train the network to your pretrain model's performance on validation set. I've visualized the distorted coordinates, I  wonder if this treatment can help training.

Have you meet such situations 1, 2 and 3 in your experiment and could you please answer questions 4 and 5?
Thanks!

**Comment by 1zb** [comment]
Actually I have similar questions. Did you get the answers?

**Comment by lzhnb** [comment]
> Actually I have similar questions. Did you get the answers?

I asked the author this question via e-mail and got the reply, so I close this question, would you like to share your e-mail? I'll share the reply with you.

**Comment by 1zb** [comment]
> > Actually I have similar questions. Did you get the answers?
> 
> I asked the author this question via e-mail and got the reply, so I close this question, would you like to share your e-mail? I'll share the reply with you.

This is my email address, 
biao.zhang@kaust.edu.sa

Thank you!

**Comment by dsc1126** [comment]
> > Actually I have similar questions. Did you get the answers?
> 
> I asked the author this question via e-mail and got the reply, so I close this question, would you like to share your e-mail? I'll share the reply with you.

Hi, I also have similar doubts. Do you mind share with me the reply? My email is dsc1126@hotmail.com.  Thank you.

**Comment by xiaodongww** [comment]
@lzhnb Hi, I am also interested in these questions. Could you share the reply with me? My email is `xiaodong.wu[at]vipl.ict.ac.cn`

Thanks a lot!

**Comment by symphonylyh** [comment]
@lzhnb Hi I'm also interested in th questions above. Can you please share the replies? My email is `hhhuang313@gmail.com`. Thanks!

**Comment by Dragon-Vae** [comment]
@lzhnb Hi, I'm also interested in these questions. Can you please share the reply? My email is jzl709065970@gmail.com . Thanks very much!

**Comment by Subury** [comment]
@lzhnb Hi, I'm also interested in these questions. Can you please share the reply? My email is [weizeqiang@bupt.edu.cn
](mailto:weizeqiang@bupt.edu.cn) . Thanks very much!

**Comment by Subury** [comment]
@lzhnb Hi, I'm also interested in these questions. Can you please share the reply? My email is [1446849325@qq.com](mailto:1446849325@qq.com) . Thanks very much!

**Comment by thinkrc** [comment]
@lzhnb @1zb  Hello! I'm also interested in these questions. Can you please share the reply? My Email is 874764134@qq.com. Thanks!
